exercises/dominoes/.meta/src/reference/java/Domino.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Objects;
+
+class Domino {
+    private int a;
```

Comment:
Since these are accessed using getLeft and getRight, naming them `left` and `right` might be more consistent 🙂